README.md

```diff
@@ -21,7 +21,7 @@ documentation](https://cranelift.readthedocs.io/en/latest/?badge=latest).
 For an example of how to use the JIT, see the [SimpleJIT Demo], which
 implements a toy language.
 
-[SimpleJIT Demo]: https://github.com/bytecodealliance/simplejit-demo
+[SimpleJIT Demo]: https://github.com/Cranestation/simplejit-demo
```

Comment:
@tschneidereit @sunfishcode Do we want to migrate simplejit-demo to the bytecodealliance too?